samples/nrf5340/smp_svr/README.rst

```diff
@@ -0,0 +1,168 @@
+.. _smp_svr_ext_xip:
+
+SMP Server Sample with external XIP
+###################################
+
+Overview
+********
+
+This sample implements a Simple Management Protocol (SMP) server that demonstrates splitting the application that it partially resides on internal flash and partially on QSPI flash.
+
+SMP is a basic transfer encoding for use with the MCUmgr management protocol.
+For more information about MCUmgr and SMP, please see :ref:`device_mgmt`.
+
+Refer to zephyr-rtos :zephyr:code-sample:`smp-svr` for information about SMP features implemented by the sample.
+
+Thes samples enables the spliting the application into intenal and external part thasnks to:
```

Comment:
Thes->These ?
"These samples demonstrate how an application can be split into internal and external part, where the internal part is executed from SoC internal storage and the external part is executed, with aid of the XIP feature a QSPI peripheral, from external memory"?